docs/src/main/sphinx/release/release-354.md

```diff
@@ -0,0 +1,56 @@
+# Release 354 (XXXXXXXX Mar 2021)
+
+## General
+
+* Preferred write partitioning is now triggered automatically when the number
+  of written partitions exceeds or is equal to
+  ``preferred-write-partitioning-min-number-of-partitions``. The number of written
+  partitions is estimated from stats, therefore CTAS or insert source tables needs to have
+  statistics collected. To forcefully enable preferred partitioning,
+  ``preferred-write-partitioning-min-number-of-partitions`` needs to be set to
+  ``0`` or ``1``. To fully disable preferred write partitioning, set
+  ``use-preferred-write-partitioning`` to ``false``.
+  ({issue}`6920`)
+* Improve performance of LEFT, RIGHT and FULL JOIN when one side of the join is known to produce a single row. ({issue}`7090`)
+  
+## Security
+
+* Fix `Oauth2` repeated authentication, after previous token expires. ({issue}`7172`)
+* Support Oauth2 authentication in Trino CLI. ({issue}`7054`)
+
+## Elasticsearch connector
+
+* Fix failure when reading single-valued fields for array types. ({issue}`7012`)
+
+## Hive connector
+
+* Fix possible S3 connection leaks when using AVRO table format ({issue}`7178`)
+* Use VARCHAR as type for all columns, for tables using CSV SerDe, when Glue metastore is in use ({issue}`7059`)
```

Comment:
```
Use VARCHAR as type for all columns of CSV SerDe tables in Glue metastore ({issue}`7059`)

Reply:
This sounds like behavior change. However, it's a Fix.

"Fix query failure when reading from a CSV table declared in Glue metastore to have a non-varchar column.
The CSV SerDe does not support any other type than `varchar`."

For more context, we have this fix for HMS for long, we didn't have it for Glue.

Reply:
@findepi, I agree with your suggestion, but it's missing something. The original phrasing says we're remapping all columns to `varchar`, but the alternative text is not clear about the implication (i.e., any column that is NOT `varchar` will be read as `varchar`, regardless of how it's declared in Glue).

Reply:
well.. the column perspective is actually varchar (from serde) no matter how it's declared in Glue

you could consider non-varchar declaration in Glue as a bug, which we're working around

Reply:
How about:

```
* Fix query failure when columns of a CSV table are declared as a type other than `varchar` in Glue metastore. Columns 
  are now interpreted as `varchar` values, instead. ({issue}`7059`)
```

Reply:
Sounds good